style.css

```diff
@@ -65,8 +61,7 @@ body {
     overflow: hidden;
     position: relative;
     margin : 0 auto;
-    top : 20px;
-    //margin-top : -50px;
+    top : 10px;
```

Comment:
I know that. I just want things to get cleaned up while somebody is working on it raher push it to a different task.